packages/shared/src/utils/index.js

```diff
@@ -17,4 +17,3 @@ export * from './getLabTestsFromLabRequests';
 export * from './numeralTranslation';
 export * from './enumRegistry';
 export * from './invoice';
-export * from './refreshChildRecordsForSync';
```

Comment:
The test was failing because of this export. Somehow this was overwriting some of the models exported from `@tupaia/database` when using them in tests. Still don't fully understand it but removing this works and using existing import `import { refreshChildRecordsForSync } from '@tamanu/shared/utils/refreshChildRecordsForSync';` should still work